python/pyarrow/tests/test_dataset.py

```diff
@@ -1425,3 +1425,13 @@ def test_feather_format(tempdir):
     write_feather(table, str(basedir / "data1.feather"), version=1)
     with pytest.raises(ValueError):
         ds.dataset(basedir, format="feather").to_table()
+
+
+def test_file_source_refcount():
+    from io import BytesIO
+    from sys import getrefcount
```

Comment:
Please put all standard imports at the top level.

Reply:
okay